packages/plugins/plugin-inbox/src/components/index.ts

```diff
@@ -5,7 +5,6 @@
 import { lazy } from 'react';
 
 export const EventsContainer = lazy(() => import('./Calendar'));
```

Comment:
what do you think about putting these into a namespace? E.g., `Types`.

Reply:
hmmm, these are components?